data_structures/src/serializers.rs

```diff
@@ -0,0 +1,572 @@
+extern crate flatbuffers;
+
+use crate::flatbuffers::protocol_generated::protocol::{
+    get_root_as_message, Address as FlatBufferAddress, AddressArgs, Command as FlatBuffersCommand,
+    GetPeers as FlatBuffersGetPeers, GetPeersArgs, IpAddress as FlatBuffersIpAddress,
+    Ipv4 as FlatBuffersIpv4, Ipv4Args as FlatBuffersIpv4Args, Ipv6 as FlatBuffersIpv6,
+    Ipv6Args as FlatBuffersIpv6Args, Message as FlatBuffersMessage, MessageArgs,
+    Peers as FlatBuffersPeers, Peers, PeersArgs, Ping as FlatBuffersPing, PingArgs,
```

Comment:
I think that `Peers` may be imported with two names here, as `Peers`and `FlatBuffersPeers` which may be confusing